guides/common/modules/proc_integrating-infoblox-dhcp.adoc

```diff
@@ -1,7 +1,10 @@
 [id="integrating-infoblox-dhcp"]
 = Integrating Infoblox DHCP
 
-Install the DHCP Infoblox provider on {ProductName}.
+If you have an Infoblox DHCP service in your network, you can integrate this service into {ProjectServer}.
```

Comment:
I think Infoblox is generally seen as an appliance. That appliance can run both DHCP and DNS (and more).
```suggestion
If you have an Infoblox appliance in your network, you can integrate this service into {ProjectServer}.
```